tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/globals.py

```diff
@@ -336,6 +338,13 @@
     "py.deduplicate": "true",
     "soma.init_buffer_bytes": 1 * 1024**3,
     "sm.consolidation.buffer_size": 3 * 1024**3,
+    "sm.mem.reader.sparse_global_order.ratio_array_data": 0.3,
```

Comment:
unrelated to thread pool counts, right?

also, worth adding a link toe config param docs for quicker reference? e.g https://tiledb-inc-tiledb.readthedocs-hosted.com/en/stable/c-api.html#_CPPv417tiledb_config_setP15tiledb_config_tPKcPKcPP14tiledb_error_t (is there a better page?)

Reply:
I don't see the value as the docs are easy to find - Google took me directly to the right page

https://docs.tiledb.com/main/how-to/configuration#configuration-parameters

that said, will add it :-)

Reply:
and yes, unrelated to thread count. This is an updated recommendation from TileDB, and controls memory allocation.

Reply:
> able to do Option+.

well, that explains it!  I don't have an Option key :-)